README.md

```diff
@@ -140,7 +140,8 @@ private void DataReceivedNormalEvent(object sender, SerialDataReceivedEventArgs
 {
     if (e.EventType == SerialData.WatchChar)
     {
-        // We have our special character at the end of the transmission
+        // A specified character was received during the transmission and a notification event was raised.
```

Comment:
was received during the transmission... 🤪 

Why not simply "The char to watch has been received"?
This code is at the event handler, therefore it has been raised. No doubts about it. Why the need to state the obvious?